unsymlink-lib

```diff
@@ -658,6 +659,9 @@ def main():
                    default='analyze')
     g.add_argument('--migrate', action='store_const', dest='action',
                    const='migrate', help='Perform the migration')
+    g.add_argument('--migrate-hardlink', action='store_const', dest='action',
```

Comment:
I'm wondering whether it wouldn't be cleaner to make it into a separate boolean (store-true) option, possibly in 'expert options' group.

Reply:
It would work — and be cleaner.
I hadn't noticed the "expert options" group, I'll adjust it.